cmd/environment_to_ini.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"os"
+	"strings"
+
+	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/setting"
+	"github.com/Unknwon/com"
+	"github.com/urfave/cli"
+	ini "gopkg.in/ini.v1"
+)
+
+// EnvironmentPrefix environment variables prefixed with this represent ini values to write
+const EnvironmentPrefix = "GITEA:"
```

Comment:
```suggestion
const EnvironmentPrefix = "GITEA__"
```

Reply:
I've seen this done before, it works for me. I think it's a bit touchy with shells sometimes cause there are some arbitrary rules. 

Double dash is ok with me. Our options are quite limited in this regard